packages/libs/configuration/resources/plugin-openapi-to-cadl.md

```diff
@@ -0,0 +1,11 @@
+# Default Configuration - TypeScript
+
+The V3 version of the Typescript Generator.
+
+```yaml $(openapi-to-cadl) && !$(legacy) && !$(v2)
+version: ^3.7.0
+
+use-extension:
+  "@autorest/openapi-to-cadl": "latest"
+try-require: ./readme.openapi-to-cadl.md
```

Comment:
Also need to update this to point to this file[ ](https://github.com/Azure/autorest/blob/main/packages/libs/configuration/resources/default-configuration.md)